tests/components/unifi/test_controller.py

```diff
@@ -175,6 +177,7 @@ async def test_controller_setup(hass):
     assert controller.site_role == SITES[controller.site_name]["role"]
 
     assert controller.option_allow_bandwidth_sensors == DEFAULT_ALLOW_BANDWIDTH_SENSORS
+    assert controller.option_allow_uptime_sensors == DEFAULT_ALLOW_UPTIME_SENSORS
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6184085)